src/public/modules/forms/admin/controllers/view-responses.client.controller.js

```diff
@@ -261,17 +261,32 @@ function ViewResponsesController(
     }
     Submissions.count(params).then((responsesCount) => {
       vm.downloadAttachmentResponsesCount = responsesCount
-      vm.confirmDownloadModal = $uibModal.open({
-        backdrop: 'static',
-        templateUrl:
-          'modules/forms/admin/views/download-all-attachments.client.modal.html',
-        scope: $scope,
-      })
+      $uibModal
+        .open({
+          backdrop: 'static',
+          resolve: { responsesCount },
+          controller: [
+            '$scope',
+            '$uibModalInstance',
+            'responsesCount',
+            function ($scope, $uibModalInstance, responsesCount) {
+              $scope.ok = function () {
+                $uibModalInstance.close()
+              }
+              $scope.cancel = function () {
+                $uibModalInstance.dismiss()
+              }
```

Comment:
i think `$scope` already has `$close` and `$dismiss` functions, so you don't even need to explicitly add `ok` and `cancel`?

> The scope associated with modal's content is augmented with:
> 
> $close(result) (Type: function) - A method that can be used to close a modal, passing a result.
> 
> $dismiss(reason) (Type: function) - A method that can be used to dismiss a modal, passing a reason.
> 
> Those methods make it easy to close a modal window without a need to create a dedicated controller.
>